tracker/templates/two_factor.html

```diff
@@ -0,0 +1,18 @@
+{%- extends "base.html" -%}
+{% block content %}
+			<h1>Two Factor</h1>
+            {% if display_qr %}
+                <img src="{{ qrcode(otp_string) }}">
+            {%endif%}  
```

Comment:
Remove trailing spaces

Reply:
please also use tabs in html and css files